scripts/publish/aws-s3.sh

```diff
@@ -31,20 +31,23 @@ function usage() {
   echo "    -b <s3 bucket>"
   echo "    -v <version>"
   echo "    -p <product name>"
+  echo "    -k <S3 key prefix>"
```

Comment:
I guess we can adhere to the `-k [S3 key prefix]` convention, where if the value is surrounded by `[]` (instead of `<>`), then its optional. I don't know how "standard" this is, but its what most node CLIs do.